android_splash_screen/analysis_options.yaml

```diff
@@ -0,0 +1,21 @@
+include: package:flutter_lints/flutter.yaml
+
+analyzer:
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+
+linter:
+  rules:
+    avoid_types_on_closure_parameters: true
+    avoid_void_async: true
+    cancel_subscriptions: true
+    close_sinks: true
+    directives_ordering: true
+    # Required because of the numbered sources in  lib/src/basics
```

Comment:
Was this file originally copied from the animations sample? If so, I don't believe lines 15-16 are necessary. The animations sample used numbers at the beginning of some of its Dart files, which this app does not.